packages/zowe-explorer/README.md

```diff
@@ -33,6 +33,8 @@ More information:
 
 ## What's new in Zowe Explorer 1.21.0
 
+> Zowe Explorer is compatible only with Theia 1.18.0 or higher.
```

Comment:
@IgorCATech should we move this above the release notes so it doesn't get removed accidentally with a new release?

Reply:
I can do that, but FYI the previous note hasn't been removed _accidentally_, I've removed it quite consciously, though I cannot remember why. Probably, I was quite sure it was not relevant anymore :D But this won't happen in the future, I'll keep an eye on that note. Does sounds reasonable?